proto/frequenz/api/common/v1/market/price.proto

```diff
@@ -0,0 +1,52 @@
+// Frequenz definitions of price for electricity trading.
+//
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
```

Comment:
This file needs a package definition. Looking at the file structure, I think it would be
```proto
package frequenz.api.common.v1.market;
```